workflow/scripts/plot-pca.py

```
@@ -0,0 +1,81 @@
+        )
+    )
+
+    chart = (
+        alt.layer(points, text_background, text)
+        .add_params(point_selector)
+        .interactive()
+    )
+    return chart
+
+
+df = pd.read_csv(snakemake.input[0], sep="\t")
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)